app/components/stepper.js

```diff
@@ -54,6 +54,8 @@ export default class StepperComponent extends Component {
       localStorage.setItem('first_name', this.login.userData.first_name);
       localStorage.setItem('last_name', this.login.userData.last_name);
       this.incrementStep();
+    } else {
+      alert('You must be logged in to continue');
```

Comment:
Another small UX improvement. The button click was silently failing when users were not logged in

Reply:
Nice. I really thought the page was just broken.

Reply:
Wait, just read the description. I thought this was a fix that hitting the "Start" button does nothing if you are not logged in, and was fixing that.

Reply:
Video: https://youtu.be/d5f0NCNOLvg

Reply:
@entropyReverser That doesn't require a fix as the workaround is to log in using the "Sign in with GitHub" button on top-right
![CleanShot 2024-08-16 at 17 06 38@2x](https://github.com/user-attachments/assets/107925f0-28dd-4440-aaeb-4e07c5304532)

Reply:
Yeah. I know, but I would have liked an alert like this of  "Please log in to continue"

Reply:
@entropyReverser That's exactly what this UX adds :)